test/units/TEST-75-RESOLVED.sh

```diff
@@ -1451,10 +1451,10 @@ testcase_15_wait_online_dns() {
 
     # Begin systemd-networkd-wait-online --dns
     systemd-run -u "$unit" -p "Environment=SYSTEMD_LOG_LEVEL=debug" -p "Environment=SYSTEMD_LOG_TARGET=journal" --service-type=exec \
-        /usr/lib/systemd/systemd-networkd-wait-online --timeout=20 --dns --interface=dns0
+        /usr/lib/systemd/systemd-networkd-wait-online --timeout=0 --dns --interface=dns0
```

Comment:
(Not necessary to address that in this PR, but we should accept infinite and internally uses USEC_INFINITY for timeout. We already have parse_sec_fix_0() for that.)